perf/lua/CMakeLists.txt

```diff
@@ -54,13 +54,19 @@ create_perf_lua_test(NAME gh-7089-vclock-copy)
 create_perf_lua_test(NAME uri_escape_unescape)
 
 include_directories(${MSGPUCK_INCLUDE_DIRS})
+include_directories(${EXTRA_BOX_INCLUDE_DIRS})
```

Comment:
Here I include [`third_party` from tarantool-ee](https://github.com/tarantool/tarantool-ee/blob/0a3e3d16754058a95dd56316650c68d8ed50fb73/CMakeLists.txt#L64). Not sure how to do it more correctly.

Reply:
I don't like including extra box directories. Let's define `ARROW_INCLUDE_DIRS` in Tarantool EE.

Reply:
Removed this line and included `third_party/arrow/abi.h` into CE.